packages/node/jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  modulePathIgnorePatterns: ['<rootDir>/build'],
```

Comment:
This was to silence a warning about the build modules having the same file names